### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by mathuin** [terraform-providers/terraform-provider-digitalocean#278]
### Terraform Version
```
Terraform v0.12.6
+ provider.digitalocean v1.5.0
+ provider.kubernetes v1.8.1
```

### Panic Output
https://gist.github.com/mathuin/ec57f48d8ff21cede4cf12e27bd8aa63

The panic at about line 4195 is in terraform-provider-digitalocean.  

If a zip file of my Terraform repo is desirable, please let me know and i'll figure out how to get it to you.

**Comment by andrewsomething** [comment]
Thanks for the report. This looks like the same panic reported in #272 and fixed in #273. I'll work on cutting a new release to get this fix available.

**Comment by andrewsomething** [comment]
This fix was released as [version 1.6.0](https://github.com/terraform-providers/terraform-provider-digitalocean/blob/master/CHANGELOG.md#160-august-05-2019). You can upgrade with:

    terraform init -upgrade
